index.html

```diff
@@ -0,0 +1,113 @@
+        </nav>
+    </header>
+    <main>
+        <section id="intro">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <form>
+                <input type="email" name="email" id="email" placeholder="Enter your email address">
+                <input type="submit" value="Get Started">
+            </form>
+        </section>
+        <section id="features">
+            <div class="content">
+                <img src="fire-icon-png.png" alt="fire-icon-png">
+                <div class="desc">
+                    <h2>Premium Materials</h2>
+                    <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+                </div>
+            </div>
+            <div class="content">
+                <img src="shipping.png" alt="shipping">
+                <div class="desc">
+                    <h2>Free Shipping</h2>
+                    <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+                </div>
+            </div>
+            <div class="content">
+                <img src="batterie.png" alt="batterie">
+                <div class="desc">
+                    <h2>Quality Assurance </h2>
+                    <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+                </div>
+            </div>
+        </section>
+        <section id="how-it-works">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </section>
+    </main>
+    <section id="row">
+        <div class="column">
+            <div class="card">
+                <div class="tit">
+                    <h3>TENOR TROMBONE</h3>
+                </div>
+                <h4>$600</h4>
+                <p>Lorem ipsum<br>Lorem ipsum<br>Lorem ipsum dolor<br>Lorem ipsum<br></p>
+            </div>
+            <button class="btn">Select</button>
+        </div>
+
+        <div class="column">
+            <div class="card">
+                <div class="tit">
+                    <h3>TENOR TROMBONE</h3>
+                </div>
+                <h4>$300</h4>
+                <p>Lorem ipsum<br>Lorem ipsum<br>Lorem ipsum dolor<br>Lorem ipsum<br></p>
+
+            </div>
+            <button class="btn">Select</button>
+        </div>
+
+        <div class="column">
+            <div class="card">
+                <div class="tit">
+                    <h3>TENOR TROMBONE</h3>
+                </div>
+                <h4>$900</h4>
+                <p>Lorem ipsum<br>Lorem ipsum<br>Lorem ipsum dolor<br>Lorem ipsum<br></p>
+            </div>
+            <button class="btn">Select</button>
+        </div>
+
+
+    </section>
+
+
+    <div class="footer">
```

Comment:
Why don't you use HTML tags instead of div? that would improve the Html structure and make the page more accessible. 
I think it would be better to use "footer" rather than "div"